Xamarin.Forms.Platform.iOS/Extensions/LabelExtensions.cs

```diff
@@ -93,7 +95,8 @@ public static void RecalculateSpanPositions(this NativeLabel control, Label elem
 					yaxis += (float)lineHeight;
 				}
 
-				((ISpatialElement)span).Region = Region.FromLines(lineHeights.ToArray(), finalSize.Width, startRect.X, endRect.X, startRect.Top).Inflate(10);
+				((ISpatialElement)span).Region = Region.FromLines(lineHeights.ToArray(), finalSize.Width, startRect.X, endRect.X, startRect.Top)
+					.Inflate(10);
```

Comment:
I kinda prefer the no change here tbh :p

Reply:
I do these mainly because @StephaneDelcroix  loves them